### ISSUE TITLE
Open Targets therapeutic area: Pregnancy, puerperium and perinatal conditions

### ISSUE THREAD

**Issue by paolaroncaglia** [EBISPOT/efo#490]
Proposed TA name: Pregnancy, puerperium and perinatal conditions
Current EFO TA name: No TA name exists 
Priority for OT: High
Justification: Pregnancy and anti-natal events should be clearly identified and not be combined with other reproductive disorders. Specifically for clinical trials, identifying possible events during pregnancy or shortly after birth is important for safety assessments to better understand how the (unborn) child would be affected by a certain drug or vaccine.

Pregnancy = The period from conception to birth. After the egg is fertilized by a sperm and then implanted in the lining of the uterus, it develops into the placenta and embryo, and later into a fetus. Pregnancy usually lasts 40 weeks, beginning from the first day of the woman's last menstrual period, and is divided into three trimesters, each lasting three months. https://medical-dictionary.thefreedictionary.com/pregnancy
Puerperium =  the period from the end of the third stage of labor until involution of the uterus is complete, usually lasting between 3 and 6 weeks. https://medical-dictionary.thefreedictionary.com/puerperium
Perinatal = relating to the period shortly before and after birth; from the twentieth to twenty-ninth week of gestation to 1 to 4 weeks after birth. https://medical-dictionary.thefreedictionary.com/perinatal

Currently in EFO - Pregnancy and puerperal disorders are sitting under reproductive processes. Perinatal disorders only exist as ‘specific’ types of disorders, but not as a generic term
[OLS views of 'pregnancy' and 'puerperal disorder']

ACTION: 
Create new term named Pregnancy, puerperium and perinatal conditions
Create 3 groups under this new TA of 
Pregnancy (currently in EFO)
Puerperal disorders (currently in EFO)
Perinatal conditions underneath (not in EFO. create term)

**Comment by paolaroncaglia** [comment]
- [ ] Take into account discussion in #417

**Comment by paolaroncaglia** [comment]
Summary for requested OT TA Pregnancy, puerperium and perinatal conditions:

In EFO we already have:

EFO:0009682 pregnancy disorder
Subclass of female reproductive system disease

EFO:0009683 puerperal disorder
Subclass of pregnancy disorder

As discussed in https://github.com/EBISPOT/efo/issues/417, we already proposed to move pregnancy disorder away from female reproductive system disease and possibly directly under disease. We also proposed that puerperal disorder should not be a child of pregnancy disorder, but rather a sibling.

We don’t have a grouping term for generic perinatal condition (neither does MONDO), but there are already a number of specific perinatal terms in EFO. I don’t see any problem in creating a perinatal disease/condition term. But it would be helpful to know if it should refer to medical conditions in the fetus/newborn, in the mother, or both (see below).

Note that, based on the definitions that OT provided, there is a time overlap between puerperium and perinatal. My understanding is that puerperium conditions affect the mother, while perinatal conditions refer to the unborn/newly born child. Please comment or confirm.

In summary, it should be possible to create the requested TA for “Pregnancy, puerperium and perinatal conditions”.

**Comment by paolaroncaglia** [comment]
Also see https://github.com/EBISPOT/efo/issues/492#issuecomment-507671863.

**Comment by d0choa** [comment]
Comment from S.:
Whilst the word ‘perinatal’ mainly refers to the foetus/newly born child, it is occasionally also used for maternal conditions. However, I am fine with Paola’s comment considering that other ontologies mainly use ‘perinatal’ for foetal/child conditions.

**Comment by paolaroncaglia** [comment]
@d0choa and Sandra
FYI here are my suggestions for the 'perinatal' term:

- [ ] Create EFO:NEW 'perinatal condition'
subclass of 'disease'
"A condition affecting an unborn or newly born individual, where the perinatal period is defined in humans as commencing at 22 completed weeks (154 days) of gestation and ending seven completed days after birth. Other, broader definitions of perinatal period start at 20 weeks of pregnancy and end at 4 weeks post partum."
https://www.who.int/maternal_child_adolescent/topics/maternal/maternal_perinatal/en/
https://en.wikipedia.org/wiki/Perinatal_mortality

- [ ] Search EFO for any existing class that should be a subclass of EFO:NEW 'perinatal condition'

- [ ] Notify MONDO in addition to the points already listed in https://github.com/monarch-initiative/mondo/issues/652, in case they'd like to align with EFO.

**Comment by smnorthen** [comment]
@paolaroncaglia
Your proposed solution sounds fine. I will take a look at the existing terms in EFO to see if other 'perinatal' terms (e.g. with newborn in the text instead of perinatal etc) should also be included.

**Comment by paolaroncaglia** [comment]
Note for self/EFO editor: 
- [ ] EFO:1001263 'amniotic fluid embolism' ("Blocking of maternal circulation by AMNIOTIC FLUID that is forced into uterine VEINS by strong UTERINE CONTRACTION near the end of pregnancy. It is characterized by the sudden onset of severe respiratory distress and HYPOTENSION that can lead to maternal DEATH.") should be added the parent 'pregnancy disorder'.

**Comment by smnorthen** [comment]
@paolaroncaglia 
in addition to everything with 'neonatal' in the text, the following terms should also be moved to this new TA: 
Perinatal lethal hypophosphatasia
Niemann-Pick disease type C, severe perinatal form
Glycogen storage disease due to glycogen branching enzyme deficiency
fatal perinatal neuromuscular form
asphyxia neonatorum
twin-to-twin transfusion syndrome
Atelosteogenesis type II

**Comment by paolaroncaglia** [comment]
Quick notes for self:
- [ ] check if there's duplication btw 'Postpartum Thyroiditis' and 'postpartum thyroiditis'
- [ ] edit 'postpartum depression' and 'postpartum hemorrhage' to make them (also) subclasses of 'puerperal disorder'
- [ ] 'peripartum cardiomiopathy' will also need an additional parent

**Comment by paolaroncaglia** [comment]
Note for self:
- [ ] Remove Mondo mappings if necessary until Mondo aligns
- [ ] Add Mondo mappings back after Mondo release.

**Comment by paolaroncaglia** [comment]
Note for self:
- [ ] Ask Mondo to revise the following:
MONDO:0044014 postpartum thyroiditis
subclass of 'autoimmune disease of urogenital tract'
subclass of 'autoimmune thyroid disease'
Based on the definition, is there really support for references to autoimmune and urogenital tract?

**Comment by paolaroncaglia** [comment]
- [ ] Make 'amniotic fluid embolism' a subclass of (also) 'pregnancy disorder'.

**Comment by paolaroncaglia** [comment]
Update: Mondo moved 'pregnancy disorder' and 'puerperal disorder' under 'disease or disorder', as requested; the changes will be visible in their December release. When they are imported into EFO, I can complete this ticket.

**Comment by paolaroncaglia** [comment]
'pregnancy disorder' and 'puerperal disorder' are now direct subclasses of 'disease', as discussed at length and as agreed with Mondo. More edits will follow, as per tasks/unchecked boxes in previous comments.

**Comment by paolaroncaglia** [comment]
@smnorthen 
@d0choa 
I've completed the work outlined above for the "pregnancy, puerperium and perinatal conditions" therapeutic area in EFO3. Those three disorders are now direct children of 'disease' in EFO. I've asserted appropriate classes to sit under each of them (while retaining other correct parents too where applicable, except for a few Orphanet terms that may still need work, but that will be taken care of separately). The new hierarchies will be visible in the EFO release scheduled for Monda, Dec 16. If you spot any more terms in EFO that should also be children of pregnancy, puerperium or perinatal diseases, please collect them all in a list and open a new ticket. Thank you!

**Comment by smnorthen** [comment]
Great. Will do. Thanks so much @paolaroncaglia
